### ISSUE TITLE
Remove Mongoose checkout code in the backend

### ISSUE THREAD

**Issue by TobiasDeBruijn** [svsticky/constipated-koala#1109]
Most if not all of the Mongoose code has been removed from the UI, but most is still present or commented out in the backend. This should be removed.

**Comment by TobiasDeBruijn** [comment]
All data has to ve preserved, absolutely. We'll need it for the next five years due to 'Verjaaring Mongoose'. However, all controllers can be removed

**Comment by SpookyBoy99** [comment]
Would keeping a copy of the database be sufficient? Because only then all controllers can be removed. Furthermore, if this data has to be only kept on record and will not be actively used within the next five years, will it be a problem to just remove everything related to mongoose from the live database, bringing Koala one step closer to the ReignIT goals, and keep a backup of the current database stored somewhere for the Verjaaring?

**Comment by TobiasDeBruijn** [comment]
Yes a copy would be just fine. It's only queried via SQL directly, never via Koala
